### PR TITLE
feature/add-name-to-readme-file

### PR DESCRIPTION
**Pull request by Jesus95I**
-Add changes to README file

### CODE REVIEW

**Comment by parisbs** [comment]
Change the pr title please to use gitflow

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# git-sample
\ No newline at end of file
+# git-sample
+ian de jesus barragan preciado
```
Comment (parisbs) [inline README.md:2]: Change to uppercase

**Review by parisbs — CHANGES_REQUESTED** [review]

**Comment by alexisgmz12** [comment]
Mal todo!!!!!

**Review by alexisgmz12 — APPROVED** [review]
LGTM! Puedes dejar de llorar :)

**Review by parisbs — APPROVED** [review]

**Pull request merged**